src/main/java/htsjdk/samtools/reference/FastaReferenceWriterBuilder.java

```diff
@@ -35,28 +38,36 @@
 /**
  * Buider for a {@link htsjdk.samtools.reference.FastaReferenceWriter}
  * <p>
- * You can set each of the three outputs (fasta, dictionary and index) to a file or a stream.
- * by default if you provide a file to the fasta an accompanying index and dictionary will be created.
+ * You can set each of the four outputs (fasta, dictionary and index, both gzi and fai) to a file or a stream.
+ * by default if you provide a file to the fasta an accompanying set of indexes and dictionary will be created.
  * This behaviour can be controlled by {@link #setMakeDictOutput(boolean)} and {@link #setMakeFaiOutput(boolean)}
- * <p>
+ * and {@link #setMakeGziOutput(boolean)}<p>
  * The default bases-per-line is {@value FastaReferenceWriter#DEFAULT_BASES_PER_LINE}.
  * <p>
- * Setting a file or an output stream for any of the three outputs (fasta, index or dict) will invalidate the other
+ * Setting a file or an output stream for any of the four outputs (fasta, fai, gzi or dict) will invalidate the other
  * output type (i.e. setting a file output will invalidate a previous stream and vice-versa).
  * </p>
  */
 public class FastaReferenceWriterBuilder {
     private Path fastaFile;
+    private boolean gzippedFastaFile = false;
```

Comment:
Initializer unnecessary as `false` is already the value of an uninitialized boolean.
```suggestion
    private boolean gzippedFastaFile;
```